File: behavioral/python/mediator/Computer.py
from Device   import Device

class Computer(Device):

    def __init__(self, mediator):
        Device.__init__(self, mediator)
        self._mediator.setComputer(self)

    def runTest(self) ->None:
        print("Computer test started.")
        self._mediator.runComputer()

    def endTest(self) ->None:
        print("Computer test over.")

File: behavioral/Java/mediator/src/Computer.java
public class Computer extends Device{
    public Computer(Mediator mediator) {
        super(mediator);

    }

    @Override
    public void runTest() {
        System.out.println("Computer test started.");
    }

    @Override
    public void endTest() {
        System.out.println("Computer test over.");
    }
}